app/editor/src/features/admin/sources/SourceDetails.tsx

```diff
@@ -64,6 +65,12 @@ export const SourceDetails: React.FC<ISourceDetailsProps> = () => {
           tooltip="The user that manages this content"
           options={users}
         />
+        <FormikSelect
```

Comment:
This currently doesn't allow them to select None.

Reply:
I followed the example of the other two dropdowns on the page. If the value of the field is null it displays "Select...". Once it's set you can change the value but can't set it back to null. What's the best way to set it to None in React?

Reply:
I've left another comment about how to prepend an option to handle `undefined`